app/views/application/_header.html.haml

```diff
@@ -2,87 +2,83 @@
 - if flash.any?
   - flash.each do |name, msg|
     = content_tag :div, msg, class: flash_class(name)
-%nav.navbar.navbar-custom
-  .container-fluid
-    .navbar-header
-      %button.navbar-toggle.collapsed{"aria-controls" => "navbar", "aria-expanded" => "false", "data-target" => "#navbar", "data-toggle" => "collapse", :type => "button"}
-        %span.sr-only Toggle navigation
-        %span.icon-bar
-        %span.icon-bar
-        %span.icon-bar
 
-      = link_to root_path, class: 'navbar-brand' do
-        =image_tag ("logo.svg"), :alt => "fablabs.io"
+%nav.navbar.navbar-custom.navbar-expand-lg{:role => "navigation"}
+  .container-fluid
+    = link_to root_path, class: 'navbar-brand' do
+      =image_tag ("logo.svg"), :alt => "fablabs.io"
 
+    %button.mt-0.navbar-toggler{"aria-controls" => "navbar", "aria-expanded" => "false", "data-target" => "#navbar", "data-toggle" => "collapse", "aria-label" => "Toggle navigation", :type => "button"}
+      %i.fa.fa-bars.fa-lg
 
     #navbar.collapse.navbar-collapse
 
       = form_tag '/search', class: 'navbar-form navbar-left', method: 'get' do
-        .form-group
+        .form-group.container-fluid.mb-0
           - if params[:country].present?
             = hidden_field_tag 'country', params[:country]
           = text_field_tag 'query', params[:query], class: 'form-control', autocomplete: 'off', placeholder: t("views.header.search"), id: 'search-box'
 
       %ul.nav.navbar-nav
-        %li.navbar-link
+        %li.nav-item.active.navbar-link
```

Comment:
.active should only be for one item. The active one.
Should ALL of these be active?